show/main.py

```diff
@@ -1607,6 +1615,97 @@ def vlan():
     """Show VLAN information"""
     pass
 
+@vlan.command()
+@click.argument('vlanid', required=True)
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+def id(vlanid, verbose):
```

Comment:
Looks like its the same code as in `brief `and is duplicated. Can you revisit to use the existing code?